libs/utils/release_notes.md

```diff
@@ -1,5 +1,11 @@
 # ftrack Utils library release Notes
 
+## upcoming
+
+* [fix] Properly pick up and handle error messages from DCC.
```

Comment:
Where is this fix happening? This is just on the js RPC regard right? so maybe start with JS RPC; as the other feat.